packages/fetchai/connections/oef/connection.py

```diff
@@ -563,12 +568,14 @@ async def _connection_check(self) -> None:
             if not self.channel.get_state() == "connected":  # pragma: no cover
                 self.connection_status.is_connected = False
                 self.connection_status.is_connecting = True
-                logger.warning(
+                self.logger.warning(
```

Comment:
why now `logger` and not `aea_logger`?

Reply:
because we are in `OEFConnection` (whose `logger` is inherited from `Component(WithLogger)`) and it does not clash with anything unlike not `OEFChannel(OEFAgent)`.